TestProjects/UniversalGraphicsTest_Lighting/Assets/Test/Runtime/ArrayDrawer.cs

```diff
@@ -0,0 +1,50 @@
+using UnityEngine;
+
+[ExecuteInEditMode]
+public class ArrayDrawer : MonoBehaviour
```

Comment:
why duplicate, it is available here com.unity.testing.urp/Scripts/Runtime/ArrayDrawer.cs

Reply:
This was some refactoring done by URP team and then it was reverted. The revert PR is merged to my branch to make things work again:
https://github.com/Unity-Technologies/Graphics/pull/4257/commits/5b8fb9220633fffdfcf9189fbec2eb645fe6508c